.github/workflows/test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [20]
+        node: [22]
```

Comment:
Ideally we wanna run unit tests against both versions, might be too expensive though